src/client/components/LanguagePicker/LanguagePicker.tsx

```diff
@@ -71,10 +79,14 @@ export const LanguagePicker: React.FC<Props> = ({
         <React.Fragment key={linkTo}>
           {location.pathname.startsWith(`/${linkTo}/`) ? (
             <ActiveOption color={color}>{shortTitle}</ActiveOption>
-          ) : (
+          ) : performClientSideNavigation ? (
             <LinkOption color={color} to={`/${linkTo}${path}`}>
               {shortTitle}
             </LinkOption>
+          ) : (
+            <NativeLinkOption color={color} href={`/${linkTo}${path}`}>
+              {shortTitle}
+            </NativeLinkOption>
```

Comment:
nit: Wouldn't it be possible to use `as`-prop here?

```
<LinkOption color={color} to={`/${linkTo}${path}`} as={performClientSideNavigation ? undefined : 'a'}>
  {shortTitle}
</LinkOption>
```

Reply:
I guess it doesn't work for this case 😞 

> The as prop is only used by styled when it’s not forwarded to the underlying element. By default, this means that the as prop is used for html tags and forwarded for components. To change this, you can pass a custom [shouldForwardProp](https://emotion.sh/docs/styled#customizing-prop-forwarding) which returns true for 'as' to forward it or returns false for 'as' to use it and not forward it.

<img width="1109" alt="image" src="https://user-images.githubusercontent.com/19200662/166680756-ae7afbd9-0e6f-4043-8e96-bd680424bd6d.png">